### PR TITLE
Updatecompany

### PR DESCRIPTION
**Pull request by EvsWorld**

### CODE REVIEW

File: .env.default [hunk 1]
```diff
@@ -1,2 +1,2 @@
 MONGODB_URL=
-STREETFLEET_MQ_URL=http://localhost:3000
+STREETFLEET_MQ_URL=http://localhost:4000/api/v1
```
Comment (marco-esposito) [inline .env.default]: take api/v1/ out

File: app.js [hunk 2]
```diff
@@ -53,10 +54,11 @@ app.use(async (ctx, next) => {
   try {
     decoded = jwt.verify(token, "$secretword");
   } catch (e) {
-    return await next();
+      return await next();
   }
   ctx.company = await Company.findOne({username: decoded.username});
-  await next();
+  // console.log('ctx.company: ', ctx.company);
```
Comment (marco-esposito) [inline app.js]: remove console.log

File: app.js [hunk 3]
```diff
@@ -53,10 +54,11 @@ app.use(async (ctx, next) => {
...
+  await next(ctx);
```
Comment (marco-esposito) [inline app.js]: take out ctx from next(ctx)

File: controllers/companiesController.js [hunk 4]
```diff
@@ -88,14 +88,75 @@ exports.signIn = async ctx => {
...
+  if (incompleteBody) {
+    ctx.status = 400;
+    ctx.body = {
+      errors: [
+        'Bad Request - the request could not be understood or was missing required parameters.(incomplete body)'
+      ]
+    };
+    return;
+  }
+  // ctx.company at this stage is all the company data bc queried earlier in middleware
```
Comment (marco-esposito) [inline controllers/companiesController.js:116]: uncomment line107
remove console.log

**Review by marco-esposito — CHANGES_REQUESTED** [review]
please make the required adjustments. thanks

**Review by EvsWorld — COMMENTED** [review]
I fixed those changes Marco

**Review by marco-esposito — APPROVED** [review]

**Pull request merged**